checkov/terraform/graph_builder/local_graph.py

```diff
@@ -478,9 +495,14 @@ def update_vertex_attribute(
         vertex_index: int,
         attribute_key: str,
         attribute_value: Any,
-        change_origin_id: int,
+        change_origin_id: int | None,
         attribute_at_dest: Optional[Union[str, List[str]]],
+        transform_step: bool = False,
```

Comment:
where do you use this var? or was it something like a method inheritance with broken interface?

Reply:
in the parent, I need the signature to be identical.